scripts/lib/utils

```diff
@@ -3,10 +3,8 @@
 
 readonly PROJECTS=(admiral cloud-prepare lighthouse shipyard subctl submariner submariner-charts submariner-operator)
 readonly REPO="quay.io/submariner"
-readonly ADMIRAL_CONSUMERS=(cloud-prepare lighthouse subctl submariner submariner-operator)
-readonly SHIPYARD_CONSUMERS=(admiral lighthouse subctl submariner submariner-operator)
-readonly OPERATOR_CONSUMES=(submariner lighthouse)
-readonly SUBCTL_CONSUMES=(submariner submariner-operator cloud-prepare lighthouse)
+readonly ADMIRAL_CONSUMERS=(cloud-prepare lighthouse submariner)
+readonly SHIPYARD_CONSUMERS=(admiral)
```

Comment:
```suggestion
readonly ADMIRAL_PROJECTS=(admiral)
```

For consistency with the other lists this can be renamed.

Reply:
But `SHIPYARD_CONSUMERS` is used in the `shipyard` stage, not the `admiral` stage. It seems strange to me to have the `shipyard` stage refer to `ADMIRAL_PROJECTS`...

Reply:
Maybe it'll look weird, but it will be in-line with other stages. Branch stage at the end create PRs for shipyard, Shipyard updates Admiral, Admiral updates Projects, etc.

![image](https://github.com/submariner-io/releases/assets/51078027/73e025d2-4a22-484c-9cc9-3636b182fee6)


Regardless, I think it can be eliminated altogether.

> Alternatively it can be eliminated since its one project exactly and the stage is for that project (similar to shipyard which we don't have `SHIPYARD_PROJECTS` for).